glossary/UNIVERSAL.md

```diff
@@ -0,0 +1,7 @@
+# Universal
+
+A universal (isomorphic) application is one whose code (in this case, JavaScript) can run both in the server and the client.
```

Comment:
I think that's a good idea.  Do you want me to fix that?

On 6 December 2015 at 20:48, Hugo Giraudel <notifications@github.com> wrote:

> In glossary/UNIVERSAL.md
> <https://github.com/HugoGiraudel/SJSJ/pull/46#discussion_r46774884>:
>
> > @@ -0,0 +1,7 @@
> > +# Universal
> > +
> > +A universal (isomorphic) application is one whose code (in this case, JavaScript) can run both in the server and the client.
>
> Maybe we could add a link to isomorphic entry here? What do you think?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/HugoGiraudel/SJSJ/pull/46/files#r46774884>.
>



-- 
Kind regards,

*- Robert Smith*
*Web Developer*

-----

*W*: http://rbrtsmith.com/
*T*: +46(0)73 687 5773
*E*: rbrtsmith@googlemail.com